README.md

```diff
@@ -170,7 +172,7 @@ In the settings.json (User Settings only!) the folder icons can be associated to
 }
 ```
 
-#### Custom clones
+#### Custom folder clones
```

Comment:
```suggestion
#### Custom folder icon clones
```

Reply:
Why did you resolve this without mentioning that my suggestion should not be used here? From my point of view "Custom file icon clones" and "Custom folder icon clones" should be consistent. We're not cloning folders, but the folder icons.

Reply:
I have already accepted this change